os/include/errno.h

```diff
@@ -68,7 +68,9 @@
  ************************************************************************/
 
 #include <tinyara/config.h>
+#ifndef NXFUSE_HOST_BUILD
```

Comment:
could you let me know why this is necessary?

Reply:
@sunghan-chang: compiler.h file is TizenRT + arm arch specific code, which is under "include/tinyara/" folder path, so this file cant be used/linked for Linux-host builds(X86). Already we made this such changes to make nxfuse binary. Kindly refer to below files.

include/math.h:108
include/debug.h:71
include/tinyara/math.h:108
include/tinyara/fs/ioctl.h:126
include/tinyara/fs/fs.h:60
include/tinyara/mqueue.h:60